Firestore/Example/Tests/Integration/API/FIRSourceTests.mm

```diff
@@ -0,0 +1,648 @@
+/*
```

Comment:
File should probably be renamed `FIRFirestoreSourceTests.mm` (sorry I missed this in your sed change).

Reply:
done.